app/(candidate)/dashboard/details/components/CampaignSection.js

```diff
@@ -7,6 +7,12 @@ import PrimaryButton from '@shared/buttons/PrimaryButton';
 import { updateCampaign } from 'app/(candidate)/onboarding/shared/ajaxActions';
 import { CircularProgress } from '@mui/material';
 
+const isValidUrl = (url) => {
+  const regex =
```

Comment:
can we use this helper?
https://github.com/thegoodparty/gp-webapp/blob/develop/helpers/linkhelper.js